assets/js/modules/adsense/components/setup/SetupAccountApproved.js

```diff
@@ -85,11 +85,11 @@ export default function SetupAccountApproved() {
 			__( 'Site Kit detected AdSense code for a different account %s on your site. For a better ads experience, you should remove AdSense code that’s not linked to this AdSense account.', 'google-site-kit' ),
 			parseAccountID( existingTag )
 		);
-		uncheckedMessage = __( 'By not placing the code, AdSense will not show ads on your website unless you’ve already got some AdSense code.', 'google-site-kit' );
+		uncheckedMessage = __( 'Please note that Adsense will not show ads on your website unless you’ve already placed the code.', 'google-site-kit' );
 	} else {
 		// No existing tag.
 		showProfile = false;
-		uncheckedMessage = __( 'By not placing the code, AdSense will not show ads on your website unless you’ve already got some AdSense code.', 'google-site-kit' );
+		uncheckedMessage = __( 'Please note that Adsense will not show ads on your website unless you’ve already placed the code.', 'google-site-kit' );
```

Comment:
The "Adsense" term needs to be capitalized - I just noticed this was wrong through ACs and IB, so not a problem of the PR itself really. I'll quickly fix it.